plugins/woocommerce-blocks/assets/js/blocks/product-filters-overlay/editor.scss

```
@@ -0,0 +1,4 @@
+.wp-block-woocommerce-product-filters-overlay {
+	margin-left: 20px;
+	margin-right: 20px;
```

I believe it would be better to use the CSS variables, like the ones located in ([assets/css/abstracts/_variables.scss](https://github.com/woocommerce/woocommerce/blob/6f1a28af336bc486fa0f6af16e9f884e0e729349/plugins/woocommerce-blocks/assets/css/abstracts/_variables.scss))

```suggestion
	margin-left: $gap;
	margin-right: $gap;
```